frontend/app/helpers/aspace_form_helper.rb

```diff
@@ -237,7 +237,7 @@ def label_and_select(name, options, opts = {})
       options = ([""] + options) if opts[:nodefault]
       opts[:field_opts] ||= {}
 
-      opts[:col_size] = 4
+      opts[:col_size] = 9
```

Comment:
This expands the select dropdown to be as wide as text inputs and ensures that these long option labels are less truncated.  This maybe captured in a JIRA somewere?  But was obviously an issue with some of the new enumerations we were dealing with.